Userland/Applications/Maps/MapsMapWidget.h

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2023, Bastiaan van der Plaat <bastiaan.v.d.plaat@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include "MapWidget.h"
+
+class MapsMapWidget final : public MapWidget {
+    C_OBJECT(MapsMapWidget);
+
+public:
+    bool show_users() const { return m_show_users; }
+    void set_show_users(bool show_users)
+    {
+        m_show_users = show_users;
+        if (m_show_users) {
+            if (m_users.size() == 0) {
+                get_users();
+            } else {
+                add_users_to_map();
+            }
+        } else {
+            clear_markers();
+            clear_panels();
+        }
+    }
+
+private:
+    MapsMapWidget(Options const&);
+
+    void get_users();
+
+    void add_users_to_map();
+
+    Vector<RefPtr<Protocol::Request>> m_active_requests;
+    RefPtr<Gfx::Bitmap> m_marker_gray_image;
+    bool m_show_users { false };
+
+    struct User {
+        String nick;
+        double coordinates[2];
```

Comment:
How about `Gfx::FloatPoint` instead?

Reply:
Why not use `LatLng` directly ;)

Reply:
Ah I missed that struct in the previous PR. For future reference, we tend to never abbreviate words, so `LatitudeLongitude` or simply `GeographicCoordinates` / `GeodesicCoordinates` would have been better :-)